### PR TITLE
Beta.4 changelog updates

### PR DESCRIPTION
**Pull request by joeyaiello**
Reworked the wording to link back to a blog post and give some context around the release.

### CODE REVIEW

**Comment by joeyaiello** [comment]
Also, please feel free to squash away, I definitely messed something up with the branching ;)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,24 @@
 
 ## v6.0.0-beta.4 - 2017-07-12
 
-- Add Windows PowerShell `PSModulePath` by default on Windows. (#4132)
-- Move PowerShell to `2.0.0-preview3-25426-01` and using the .NET CLI version `2.0.0-preview2-006502`. (#4144)
+## Windows PowerShell backwards compatibility
+
+In the `beta.4` release, we've introduced a change to add the Windows PowerShell `PSModulePath` to the default $PSModulePath in PowerShell Core on Windows. (#4132)
```
Comment (daxian-dbw) [inline CHANGELOG.md]: > to the default $PSModulePath

Should be `$env:PSModulePath`

Comment (joeyaiello) [inline CHANGELOG.md]: whoops, meant it to just be `PSModulePath` (the `$env:` is too confusing). Changing now.

Comment (joeyaiello) [inline CHANGELOG.md]: Fixed.

**Review by daxian-dbw — APPROVED** [review]
LGTM

**Pull request merged**